activemodel/test/models/post.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Post
+  include ActiveModel::Model
```

Comment:
Note for myself:
Presumably it would be more suitable to include `ActiveModel::API` instead of `ActiveModel::Model`